prqlc/packages/snap/snapcraft.yaml

```diff
@@ -17,8 +17,14 @@ confinement: strict
 icon: web/website/static/img/icon.svg
 
 parts:
+  rust-deps:
+    plugin: nil
+    override-pull:
+      curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+      --profile minimal --default-toolchain 1.69.0
```

Comment:
Can we remove any hardcoding and always grab the latest? Rust is fine with a _later_ compiler...